mobile/src/components/common/input/styles.ts

```diff
@@ -35,6 +35,7 @@ const styles = StyleSheet.create({
   },
   rows: {
     textAlignVertical: 'top',
+    maxHeight: 96,
```

Comment:
maybe it's better to restrict setRowsCount, like `messageChar === '\n' && rowsCount < ROWS_MAX_COUNT`